### PR TITLE
doc: update Zigbee mentions in docs

### PR DESCRIPTION
**Pull request by greg-fer**
Update mention of Zigbee in nRF52 UG.
Highlight Zigbee in nRF5340 UG.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_nrf52.rst [hunk 1]
```diff
@@ -90,8 +90,7 @@ The nRF52 Series multi-protocol radio supports Bluetooth Low Energy, proprietary
 Standard interface protocols like NFC and USB are supported on a range of the devices in the series and with supporting software.
 
 .. note::
-   |NCS| currently has limited support for Thread.
-   It does not support ANT or Zigbee.
+   |NCS| currently has no support for ANT.
```
Comment (greg-fer) [inline doc/nrf/ug_nrf52.rst:93]: - [ ] Verify support for ANT

File: doc/nrf/ug_nrf52.rst [hunk 2]
```diff
@@ -151,6 +150,15 @@ See :ref:`zephyr:usb_api` for documentation and :ref:`zephyr:usb-samples` for a
...
 
+Zigbee
+======
+
+.. include:: ug_zigbee_overview.rst
+   :start-after: zigbee_ug_intro_start
+   :end-before: zigbee_ug_intro_end
+
+See the :ref:`Zigbee user guide <ug_zigbee>` for information about how to work with the Zigbee protocol and stack.
+To start developing, check out the :ref:`Zigbee light switch <zigbee_light_switch_sample>` sample.
```
Comment (ru-fu) [inline doc/nrf/ug_nrf52.rst:162]: Will add a section for Thread.

File: doc/nrf/ug_nrf5340.rst [hunk 3]
```diff
@@ -151,7 +151,10 @@ For IEEE 802.15.4, the nRF Connect SDK makes use of Zephyr's :ref:`zephyr:nrf-ie
 This Zephyr sample is designed specifically to enable the nRF IEEE 802.15.4 radio driver and its serialization library on a remote MCU (for example, the nRF5340 network core) using the `RPMsg Messaging Protocol`_ as a transport for the nRF 802.15.4 radio driver serialization.
 The sample implements the RPMsg transport using the `OpenAMP`_ library to communicate with the nRF IEEE 802.15.4 radio driver serialization host that runs on a separate core (for example, the nRF5340 application core).
 
-This sample must be programmed to the network core to run samples for 802.15.4-based protocols like Thread or Zigbee on nRF5340.
+Thread and Zigbee
```
Comment (ru-fu) [inline doc/nrf/ug_nrf5340.rst:154]: I don't think this should be a subsection. Would there be any other subsections here?
I like Krzysztof's suggestion to add Thread and Zigbee to the IEEE 802.15.4 heading.

File: doc/nrf/ug_nrf52.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/ug_nrf52.rst:93]: Verified support status with Joel. The current wording is OK.

File: doc/nrf/ug_nrf5340.rst (hunk 3 above)
Comment (greg-fer) [reply doc/nrf/ug_nrf5340.rst:154]: What about "IEEE 802.15.4 (Thread and Zigbee)"?
I'm not a fan of backslashes in headings, and this option allows us to use subsections if more protocols start using 802.15.4.

Comment (ru-fu) [reply doc/nrf/ug_nrf5340.rst:154]: Sure, sounds good.

**Comment by ru-fu** [comment]
Added a section for Thread and cleaned up the information about multiprotocol.

**Review by MarekPorwisz — APPROVED** [review]
Seems to cover all Krzysztofs comments.

**Review by tomchy — APPROVED** [review]

**Review by ru-fu — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**